src/styles/index.scss

```diff
@@ -1,3 +1,126 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  flex-direction: column;
+  width: 200px;
+  margin: 0 auto;
+  padding: 32px 16px 16px;
+  display: flex;
+}
+
+.card__top {
+  width: 160px;
+  height: 134px;
+  padding: 0 3px;
+}
+
+.card__imac {
+  width: 160px;
+  height: 134px;
+}
+
+.card__bottom {
+  flex-direction: column;
+  margin-top: 40px;
+  display: flex;
+}
+
+.card__title {
+  color: #060b35;
+  margin-bottom: 4px;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  display: block;
+}
+
+.card__subtitle {
+  color: #616070;
+  margin-bottom: 16px;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.card__quality {
+  justify-content: space-between;
+  align-items: center;
+  display: flex;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(../images/star.svg);
+  background-position: 50%;
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars--4 .stars__star:nth-child(-n + 4) {
```

Comment:
you should copy all modifiers from previous task

Reply:
Not fixed

Reply:
> Not fixed


I'm sorry, I didn't quite understand what I need to fix here

Reply:
Please, add a possibility to change the stars quantity by changing the class with a modifier